posthog/management/commands/resync_persons_on_events.py

```diff
@@ -0,0 +1,131 @@
+# based of 0006_persons_and_groups_on_events_backfill.py
+# because we can't easily import sth that starts with a number :(
+import importlib
+
+import structlog
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from posthog.async_migrations.utils import execute_op_clickhouse
+
+mig_0006_mod = importlib.import_module("posthog.async_migrations.migrations.0006_persons_and_groups_on_events_backfill")
+
+logger = structlog.get_logger(__name__)
+
+
+def run_resync(options):
+    if not options["team_id"]:
+        logger.error("You must specify --team-id to run this script")
+        exit(1)
+
+    TEAM_ID = options["team_id"]
+    QUERY_ID = f"Persons on Events resync team {TEAM_ID}"
```

Comment:
This won't be parseable in the query log.

```suggestion
    QUERY_ID = f"resync_persons_on_events:{TEAM_ID}"
```